aips/passkeys.md

```diff
@@ -0,0 +1,504 @@
+ 
+**[`AuthenticatorSelectionCriteria`](https://www.w3.org/TR/webauthn-3/#dictdef-authenticatorselectioncriteria)**
+
+This field defines additional information about the credential and provides constraints that affect the UX of registering / asserting with the credential. See comments above the fields below for more info:
+
+```js
+dictionary AuthenticatorSelectionCriteria {
+  // Can be "platform" | "cross-platform"
+  // Authenticators can either be "roaming" or "platform" authenticators
+  // Roaming authenticators (like phones or yubikeys) can be used to 
+  // authenticate a user via Bluetooth or QR Code
+  DOMString                    authenticatorAttachment;
+  // Can be "required" | "preferred" | "discouraged"
+  // This value indicates the Relying Party requires 
+  // a client-side discoverable credential, and is 
+  // prepared to receive an error if a client-side 
+  // discoverable credential cannot be created.
+  DOMString                    residentKey;
+  boolean                      requireResidentKey = false;
+  // Can be "required" | "preferred" | "discouraged"
+  // If required, it indicates that the RP requires 
+  // user verification for the operation
+  DOMString                    userVerification = "preferred";
+};
+```
+
+#### Registering WebAuthn Credential
+
+In order to perform a [registration ceremony](https://www.w3.org/TR/webauthn-3/#registration-ceremony), the [Relying Party](https://www.w3.org/TR/webauthn-3/#relying-party) MUST proceed as follows:
+
+1. Let `options` be an instance of [`PublicKeyCredentialCreationOptions`](https://www.w3.org/TR/webauthn-3/#dictdef-publickeycredentialcreationoptions), configured to the [Relying Party](https://www.w3.org/TR/webauthn-3/#relying-party)'s needs for the ceremony.
+2. Call [`navigator.credentials.create()`](https://w3c.github.io/webappsec-credential-management/#dom-credentialscontainer-create) and pass options as a parameter to the function.
+
+#### Registration Response
+
+After successfully calling [`navigator.credentials.create()`](https://w3c.github.io/webappsec-credential-management/#dom-credentialscontainer-create), the authenticator will return a [`PublicKeyCredential`](https://www.w3.org/TR/webauthn-3/#publickeycredential). In the `PublicKeyCredential` will be a `response` field of type [`AuthenticatorAttestationResponse`](https://www.w3.org/TR/webauthn-3/#authenticatorattestationresponse) which will contain most of the information needed for verification of the registration response.
```

Comment:
Again, not sure that this level of detail is at all necessary.

Reply:
Removed the `Registering WebAuthn Credential` section